chcemvediet/templates/main/homepage/about.html

```diff
@@ -14,7 +14,7 @@
   <div class="col-md-4 col-sm-6 chv-justify">
     {% include "main/homepage/texts/about.html" %}
     <div class="text-center chv-above-1">
-      <a class="btn chv-btn-blue" href="{% page "/o-projekte/" %}">
+      <a class="btn chv-btn-blue" href="{% page '/o-projekte/' %}">
```

Comment:
@martinmacko47 Interpreter mal problem zobrat vnorene zatvorky. Prerobil by som to aj na ostatnych miestach v kode

Reply:
Kludne. Len na ostatnych miestach pls v separatnom PR potom.